### ISSUE TITLE
2024_07 #story_b: Test dynamic decision dashboard wireframe & alt text for 508 compliance

### ISSUE THREAD

**Issue by ljmoody** [lzim/teampsd#3198]
## AC:
- [ ] all DDD elements are identified, tagged, have alt text/audio description/descriptive narration and corresponding visual
- [ ] missing items are highlighted in yellow, identified by Tejas' team 6/28/2024:
(DEV has already implemented the complete tab order on the sim page of the model)
![image (1)](https://github.com/lzim/teampsd/assets/124643236/992fcfda-4b60-4023-bfb9-c3e52bbf3df7)
- [ ] missing items are added to the 508 wireframe
- [ ] DEV incorporates alt text/audio description/descriptive narration for missing items
- [ ] tab order is tested and matches the tag # and correct description

**Comment by ljmoody** [comment]
Tag # | Function | Description
--- | --- | ---
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 | 
1 |

**Comment by ljmoody** [comment]
1 |

**Comment by ljmoody** [comment]
@tejaspesquare, I added new **YELLOW** tags to your screenshot to identify and order the missing elements. Please note: 
- I added number 29, an additional "i" icon not previously tagged.  
- An additional screenshot of Team Flow (not in your screenshot, but likely needs additional alt text)

![image](https://github.com/lzim/teampsd/assets/124643236/562e6097-81f0-4f7b-86bb-b4dba17aa697)

![image](https://github.com/lzim/teampsd/assets/124643236/ec161c63-9bfd-44f3-8004-76c52dd27339)

**YELLOW** Tag # | Alt Text / Audio Description / Descriptive Narration
:---:  | :--- 
1 | Experiment window caret. Click to contract and expand the window.
2 | Team Data window caret. Click to contract and expand the window.
3 | "i" icon. Click for a pop-up with more information about the labeled feature.
4 | "i" icon. Click for a pop-up with more information about the labeled feature.
5 | "i" icon. Click for a pop-up with more information about the labeled feature.
6 | "i" icon. Click for a pop-up with more information about the labeled feature.
7 | Care Coordination services button. Click to see stock values for Care Coordination.
8 | Medication Management services button. Click to see stock values for Medication Management.
9 | Psychotherapy services button. Click to see stock values for Psychotherapy.
10 | Evidence-based Psychotherapy services button. Click to see stock values for Evidence-based Psychotherapy.
11 | Adjunctive services button. Click to see stock values for Adjunctive.
12 | Group Therapy services button. Click to see stock values for Group Therapy.
13 | "i" icon. Click for a pop-up with more information about the labeled feature.
14 | "i" icon. Click for a pop-up with more information about the labeled feature.
15 | "i" icon. Click for a pop-up with more information about the labeled feature.
16 | "i" icon. Click for a pop-up with more information about the labeled feature.
17 | Depression meds needed radio button. Click to see stock values for Depression.
18 | Opioid Use Disorder meds needed radio button. Click to see stock values for Opioid Use Disorder.
19 | Other meds needed radio button. Click to see stock values for Other.
20 | "i" icon. Click for a pop-up with more information about the labeled feature.
21 | "i" icon. Click for a pop-up with more information about the labeled feature.
22 | "i" icon. Click for a pop-up with more information about the labeled feature.
23 | "i" icon. Click for a pop-up with more information about the labeled feature.
24 | "i" icon. Click for a pop-up with more information about the labeled feature.
25 | "i" icon. Click for a pop-up with more information about the labeled feature.
26 | "i" icon. Click for a pop-up with more information about the labeled feature.
27 | "i" icon. Click for a pop-up with more information about the labeled feature.
28 | "i" icon. Click for a pop-up with more information about the labeled feature.
29 | "i" icon. Click for a pop-up with more information about the labeled feature.
30 | Specialty Mental Health setting radio button. Click to see stock values for Specialty Mental Health.
31 | Primary Care/Primary Care Mental Health Integration setting radio button. Click to see stock values for Primary Care/Primary Care Mental Health.
32 | "i" icon. Click for a pop-up with more information about the labeled feature.
33 | "i" icon. Click for a pop-up with more information about the labeled feature.
34 | "i" icon. Click for a pop-up with more information about the labeled feature.
35 | "i" icon. Click for a pop-up with more information about the labeled feature.
36 | "i" icon. Click for a pop-up with more information about the labeled feature.
37 | "i" icon. Click for a pop-up with more information about the labeled feature.
38 | "i" icon. Click for a pop-up with more information about the labeled feature.
39 | "i" icon. Click for a pop-up with more information about the labeled feature.

In addition, the following **RED** tag numbers & alt text couplings need to be updated:

Tag | Alt Text
 :---: | :---
43 | Intake services radio button. Click to see stock values for Intake.
53 | Alcohol Use Disorder meds needed radio button. Click to see stock values for Alcohol Use Disorder.
69 | General Mental Health setting radio button. Click to see stock values for General Mental Health.

fyi @jamesmrollins @lijenn

**Comment by ljmoody** [comment]
@tejaspesquare, is there an update for this issue card? Looking for a time when I can test for the additional yellow tags identified above.

**Comment by tejaspesquare** [comment]
Hello @ljmoody, 

The task has yet to begin as the Designer was involved in designing Team Care & Team Flow tasks. As previously stated, he has been preoccupied with some tasks at the organizational level for a few days. Upon his return, we will promptly notify him of outstanding requirements.

**Comment by tejaspesquare** [comment]
hello @ljmoody, @lijenn, @matomasz

This development can commence only after all requirements have been finalized and all pertinent development tasks have been completed. Could you please confirm whether we anticipate any changes specifically to the UI elements that would impact the order of elements following the development phase?

Thank You!

**Comment by matthewtomo** [comment]
**week 3 2024_08 Praxis Meeting**
@matthewtomo @jamesmrollins @ljmoody 

- @tejaspesquare We do not expect any upcoming changes to this page. You have the go ahead for finalizing ordering and alt text.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_2 2024_9**
@matthewtomo @tejaspesquare 

- Current alt text for first link differs from what is available in the wireframe and what is available in the legacy wireframe
  - @ljmoody see email with subject: "508 Compliance Confirmation" from @tejaspesquare 
- Which version should Dev team be using for the alt text?
- Additionally, do we have wireframes for the simulation pages?

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk_3 2024_9**
@matthewtomo @tejaspesquare 

- @ljmoody, Dev team has a few questions about this card (see also email with subject: "508 Compliance Confirmation")
  - [ ] The current alt text for first link differs between what is available in the wireframe and what is available in the legacy wireframe. Which should they be using?
    - Current wireframe: ![image](https://github.com/user-attachments/assets/f3f91ed7-6a20-47a9-befc-a1f4c063a861)
    - Legacy wireframe: ![image](https://github.com/user-attachments/assets/b64b892f-7f07-460b-98d0-21c752812765)
- I would think current wireframe, but want to verify with you.

**Comment by ljmoody** [comment]
@matthewtomo and @tejaspesquare, after reviewing the legacy wireframe and the one I created, a **hybrid of these two documents** may offer the best solution. 

1. _MTL_ logo at top left of screen: Yes, please include alt text for this. 
   - Revise alt text to say "_MTL_ logo. Click this logo on any page to return to the team Home page."
   - That said, this raises what may become a new issue card/assigned task, because clicking this icon does not elicit the same behavior -- neither between pages under one type of login, nor between types of logins:

| Page in Sim where _MTL_ logo is clicked | Administrator login reaction | Field User login reaction |
|:--|:--|:--|
| Admin Dashboard | no change (but reloads page) | n/a |
| Community | no change | n/a |
| Play (main Sim UI Play page) | team Home page | team Home page |
| Team (chat) | Admin Dash | team Home page |
| More Info | no change | team Home page |
| Tutor | no change | team Home page |
| Tips | no change | n/a |
| _MTL_ Demo | logo change to demo, no change (stays in demo) | n/a |
| Team PSD | logo changes, no change, stays in VA Team PSD | n/a |
| Report Bug | -- | -- |
| News | -- | -- |

   - **Suggest making the clickable mtl logo navigate to the team Home page - from any page + with any user type login.** _Can you think of a reason not to?_ It's possible I could be overlooking something.
     - Rationale: 1) same UX, no matter the type of login and 2) Admins have "Admin Dashboard" icon at top navigation bar from any page.
     
2. Otherwise, recommend using the wireframes I developed for the Dynamic Decision Dashboard:
   - [wireframe_DDD_508_narration](https://takouba-my.sharepoint.com/:b:/g/personal/lmoody_praxisanalytix_com/ETl7EzLM_bBFtfiengwXpOEB_Oyx2qq9xpvOicJsAr-eng?e=6PkVob)
   - Additional 508 callouts [in this comment](https://github.com/lzim/teampsd/issues/3198#issuecomment-2201131164)
   
3. Re: the Sim UI Play page, this is also considered a product of the government. So yes, we need 508 compliance--and alt text. 
   - Suggest using James' [508 legacy wireframe .pdf](https://takouba-my.sharepoint.com/:b:/g/personal/lmoody_praxisanalytix_com/EX_zRxr7nqBGiunDOWGZ6usBkxEVFLsQRMKPwStuKIOFcQ?e=f7Slrn) as an initial wireframe for this task.
   - If there are missing elements discovered while developing the alt text for the Sim UI Play page, please let me know and I will track those down.
   
4. From Tejas' 9/13/24 email:
![image](https://github.com/user-attachments/assets/24fc9cf7-88a3-4744-8453-ed34ff3baba4)
   - The highlighted alt text descriptions can be found in the "Secondary Pop-ups" section of the wireframe_DDD_508_narration file (linked above in point 2). 
   - I checked the wireframe and although it doesn't specifically cover Team Care module's Episodes of Care (highlighted above), please use the alt text for Episodes of Care from any other module (it's all the same) and that will do.

**Comment by tejaspesquare** [comment]
Hello @ljmoody , @matomasz,

Below is a summary following our discussion with the dev and design teams. Required you attention on ii, iii & iv.

i. We will utilize the updated alt text for the _MTL_ logo.
ii. The top navigation links (Home, Play, More Info, etc.) will be required additional work if we are to adhere to the most recent wireframe, as this task need to replicate across pages (Dashboard/Play based on Login).  
iii. Regarding the Home icon and others, we propose that instead of referring to it as an 'icon,' we should designate it as a 'Button.'
iv. From above Point no 4: Feedback Loops - if there is a specific sequence we need to follow within the loop, please share your guidelines.
![image](https://github.com/user-attachments/assets/7c5217d3-0ce6-491a-a8ea-60ecec976743)
Likewise, Decision Meter & Episodes of Care too required some inputs to maintain order.
v. SIM UI Play Page- We shall refer to legacy wireframe '508_instructions_Legacy.pdf'.

**Comment by matthewtomo** [comment]
**Discussed support_workflow wk_3 2024_09**
@matthewtomo @ljmoody 

- @tejaspesquare, we have a few questions and recommendations based on your last comment.
- For point ii, what would be the additional work required for following the most recent wireframe? It does not look like there is currently alt text implemented for the navbar in the DEV environment for either the sim_ui or the team-psd website. Is there some other consideration being made besides differences in alt text between the two wireframes?
- For point iii, in order to maintain consistency with the manual, we would prefer to continue referring to the clickable elements of the navbar as icons.
- For point iv, @ljmoody and I are not currently aware of any wireframes covering alt text or tab order for the CLDs, Flow Diagrams, Decision Meters, or episodes of care. Before making any major alterations to the current scope, would like to loop in @jamesmrollins to see if there are already designed wireframes for this purpose, or if it will be a task for the upcoming roadmap.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_3 2024_09**
@matthewtomo @tejaspesquare 

- We noted that the scope of further wireframes may be better handled at the upcoming Roadmap Planning Session.
- Development has been started based on the above discussion for existing wireframes (From @ljmoody and @jamesmrollins )

**Comment by matthewtomo** [comment]
**Discussed Monday Dev Standup wk_4 2024_09**
@matthewtomo @tejaspesquare 

- Development is still ongoing.
- When an estimate for the remaining task time is available, @tejaspesquare will be able to provide that estimate here.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_4 2024_09**
@matthewtomo @tejaspesquare 

- Designer work is almost finished
- Once finished, Designer will need to collaborate with Dev to implement final changes
- Development is expected to be complete in two weeks.
  - Note, development has expanded to include more pages than the DDD based on @ljmoody's above comment

> * Re: the Sim UI Play page, this is also considered a product of the government. So yes, we need 508 compliance--and alt text.
>   
>   * Suggest using James' [508 legacy wireframe .pdf](https://takouba-my.sharepoint.com/:b:/g/personal/lmoody_praxisanalytix_com/EX_zRxr7nqBGiunDOWGZ6usBkxEVFLsQRMKPwStuKIOFcQ?e=f7Slrn) as an initial wireframe for this task.
>   * If there are missing elements discovered while developing the alt text for the Sim UI Play page, please let me know and I will track those down.
>
~ https://github.com/lzim/teampsd/issues/3198#issuecomment-2357169548

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_1 2024_10**
@matthewtomo @tejaspesquare 

- Design is almost finished for DDD, development is in progress
  - The Learning Mode Popup still needs to be developed further to support tab ordering
- Final 508 compliance for DDD and simulation/play pages is expected to be finished by the 3rd week of October

fyi: @ljmoody

**Comment by ljmoody** [comment]
@matthewtomo, _is it possible for the mtl logo link to be in place for by wk4 October?_ 

- Need to know for inclusion in the Sim UI Cheatsheet (example given below-highlighted portion) & MTL Manual.
- Field user experience looks good to go, but Admin login has a different UX. 
- It's related to a [comment above](https://github.com/lzim/teampsd/issues/3198#issuecomment-2357169548); I've only pasted in the part relevant to this ask.
---
1. _MTL_ logo at top left of screen: Yes, please include alt text for this. 
   - Revise alt text to say "_MTL_ logo. Click this logo on any page to return to the team Home page."
   - That said, this raises what may become a new issue card/assigned task, because **clicking this icon does not elicit the same behavior -- neither between pages under one type of login, nor between types of logins**:

| Page in Sim where _MTL_ logo is clicked | Administrator login reaction | Field User login reaction |
|:--|:--|:--|
| Admin Dashboard | no change (but reloads page) | n/a |
| Community | no change | n/a |
| Play (main Sim UI Play page) | team Home page | team Home page |
| Team (chat) | Admin Dash | team Home page |
| More Info | no change | team Home page |
| Tutor | no change | team Home page |
| Tips | no change | n/a |
| _MTL_ Demo | logo change to demo, no change (stays in demo) | n/a |
| Team PSD | logo changes, no change, stays in VA Team PSD | n/a |
| Report Bug | -- | -- |
| News | -- | -- |

   - **Suggest making the clickable mtl logo navigate to the team Home page - from any page + with any user type login.** _Can you think of a reason not to?_ It's possible I could be overlooking something.
     - Rationale: 1) same UX, no matter the type of login and 2) Admins have "Admin Dashboard" icon at top navigation bar from any page.
     - **update 9/25**: The mtl.how 3.7 Sim UI Cheatsheet also references this logo and behavior. Will be updating the cheatsheet in the next week and need to know if this should stay as-is or be revised. Tracking via #3267.
![image](https://github.com/user-attachments/assets/519e309a-440e-4b88-a186-41441ce5cfab)

**Comment by lzim** [comment]
@ljmoody 

_What logo do you need?_
_It's not here? https://github.com/lzim/teampsd/tree/master/resources/logos_

I made all the original .svg .ai .psd and .png MTL and Team PSD logos and icons so they would scale and follow the VHA Style requirements.

Let me know if you're missing one and please be sure that the logos and icons are signed off on by me.
Thx!

**Comment by ljmoody** [comment]
@lzim, I understand why you think I needed a logo; that isn't what I meant. I'm speaking to the behavior of the logo (when users click on it) and some dev work associated or new explanation needed for that logo on the Sim UI Cheatsheet.

> _What logo do you need?_ _It's not here? https://github.com/lzim/teampsd/tree/master/resources/logos_
> 
> I made all the original .svg .ai .psd and .png MTL and Team PSD logos and icons so they would scale and follow the VHA Style requirements.
> 
> Let me know if you're missing one and please be sure that the logos and icons are signed off on by me. Thx!

**Comment by tejaspesquare** [comment]
Hello @ljmoody, @matthewtomo ,
I would like to clarify couple of things from James' 508 legacy wireframe. (Simulation page (Run))
 
**1.** _MTL_ logo - This is not in scope for 508 compliance on Run. should we bypass the tab order for this element? 
![image](https://github.com/user-attachments/assets/1474e61b-2ea7-4ce4-8cb0-e41ba1a737a5)

**2**. As soon you arrived on Run page focus goes to Text window. If you user wants to operate with Tab order on entire page, then current setup is not allowed user to come out from this popup. If we intend to change then please advise where should focus go.
![image](https://github.com/user-attachments/assets/be7d4aa3-1571-4fb8-b355-c8e12a0bd0c5)

**3**. <ins>Current Appearance</ins>
![image](https://github.com/user-attachments/assets/b89829d7-703c-4dc5-a934-51a88917a488)
<ins>Old</ins> - Wireframe Alt text does not guide user to click on all checkbox to get into entire flow of a model. We shall improve text message here. If it is agreeable to all, then kindly share revised details.
![image](https://github.com/user-attachments/assets/11f8ec14-dde4-49a3-b259-e3098a38d702)
![image](https://github.com/user-attachments/assets/d385a78f-c9b0-418b-8a6f-3f2be44a5459)

Thank You!

**Comment by matthewtomo** [comment]
**Discussed Dev ad_hoc**
@matthewtomo @tejaspesquare 

In general, the tab ordering for all elements on the page should be a top to bottom, and left to right (like reading a page in English)

1. I suggest not including MTL logo in tab ordering, because the logo does not have the same behavior on all pages, and I do not think that it would make sense for the logo to have the same behavior on all pages if the desired behavior on some pages is to navigate to the home page.

2. Focus should start on the team name in the header.

Additionally, Propose the inclusion of the following alt text for the two labeled elements:
![image](https://github.com/user-attachments/assets/636d850d-20b5-495e-91ef-248e44d7ad14)
- Element 1: "Outputs and Text Header - Open Test Popup (This will return you to the main area of this page)"
- Element 2: "Test Popup Header - Close Test Popup (This will return you to the main area of this page)"
3. Keep consistent with James' alt text, but edit where the order of elements has changed, and where elements have been renamed.

@ljmoody

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk2 2024_10**
@matthewtomo @tejaspesquare 

- To Be Completed
  - Learning mode pop-up
  - Team data section on Home (DDD) Page (handling scrolling)
  - Play page ordering still needs to be verified (but development is believed to be completed)

**Comment by tejaspesquare** [comment]
Hello @ljmoody , @matthewtomo,

It is important to note that linking the upload feature (Team Data) with 508 compliance is not feasible, as the control extends beyond the application.
![image](https://github.com/user-attachments/assets/eb1d99d1-2103-484f-9cf2-f254db6449de)

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk3 2024_10**
@matthewtomo @tejaspesquare 

- Upload button should have an alt text like "Click to confirm the upload and a File Explorer window will open."
- From there the user's machine's accessibility software should handle anything external to our application.

**Comment by matthewtomo** [comment]
**Discussed Ad Hoc Meeting**
@matthewtomo @jamesmrollins 

- Alt Text for the "Outputs and Text" Popup are attached here.
- It looks like they might already be implemented correctly.

fyi: @tejaspesquare 
[Expanded_Outputs_Tile_508_instructions.pdf](https://github.com/user-attachments/files/17438652/Expanded_Outputs_Tile_508_instructions.pdf)

**Comment by matthewtomo** [comment]
**Discussed Monday Dev Standup wk1 2024_11**
@matthewtomo @tejaspesquare 

- Team Data section of DDD is encountering some scrolling errors when using a screen reader.
  - This is the last 508 compliance development required for the DDD

- Development is still in progress for the following items on play page:
  - Connecting the popups to main page via tab navigation
  - tab ordering for the expanded Outputs and Text Popup

**Comment by lzim** [comment]
Great work!

**Comment by matthewtomo** [comment]
**Discussed Monday Dev Standup wk1 2024_12**
@matthewtomo @tejaspesquare 

- This card has been allowed to grow too much in scope.
- [ ] @matthewtomo will move any play page related 508 work to a new card
- This card will remain **ONLY** for work on the Dynamic Decision Dashboard
- The remaining work for the DDD before further validation testing is:

> - Team Data section of DDD is encountering some scrolling errors when using a screen reader.
>   - This is the last 508 compliance development required for the DDD

**Comment by ljmoody** [comment]
FYI @matthewtomo, tested in DEV instance today with VA laptop, Edge browser, built-in screen reader.
Screen reader correctly read all text within the Team Data section of the DDD. Highlighting corresponded with narrator voice and scrolling happened as it went down the data file list. I did not listen to the entire list, only the first 10 or so.
![image](https://github.com/user-attachments/assets/4805a86f-5b02-4140-a8a3-b6865abed7f4)

**Comment by matthewtomo** [comment]
**Discussed sim_ui wk5 2024_04**
@matthewtomo @ljmoody

- These changes are included in the current batch of issues being tested

FYI: @lzim

**Comment by lzim** [comment]
Great thanks @matthewtomo 
- _When is this expected to release to PROD?_

FYI: @lijenn

**Comment by ljmoody** [comment]
Hi @matthewtomo,
Today's testing in TEST instance revealed:
- [ ] Care Coordination module: Screen reader should allow tabbing through stock headings and "i" icons + text before Directives and More menu; it currently does not. 
![Image](https://github.com/user-attachments/assets/43ad79c6-d772-4abd-8b7b-90c539da1c0b)

- All module stocks are not read by 508 screen reader; however, the title and the "i" description IS, which I think suffices.  Otherwise, it's basically stated twice. Screenshot below to clarify what I mean.
<img width="570" alt="Image" src="https://github.com/user-attachments/assets/0d1dcf2f-6efc-4360-9207-bbd7551dc384" />

- [ ] Medication Management radio buttons are not prefaced with the overall description like other modules with radio buttons. Screenshot below of the missing reader message:
<img width="992" alt="Image" src="https://github.com/user-attachments/assets/e7ad77a3-6473-4fcc-b015-337597d8ec8a" />
 
- [ ] From the DDD>Team Data window>Upload button, removed text makes the yes/no buttons confusing; suggested improvement:
![Image](https://github.com/user-attachments/assets/298c99da-2579-45bd-88ee-691dc5eae40d)

- [ ] There are no reader messages yet for feedback loops, decision meters, flow, episodes of care. These are cards #3302, #3303, #3304, #3305. How should this be prioritized moving forward? It would take a chunk of time (for me), then dev/you, etc.

ℹ : I did not verify 508 compliance for Sim UI Play pages. I'm at my weekly limit of hours and can begin verifying those next week.

**Comment by matthewtomo** [comment]
@ljmoody Thank you for all your hard work documenting testing for this card!

> Hi @matthewtomo,
> Today's testing in TEST instance revealed:
> - [ ] Care Coordination module: Screen reader should allow tabbing through stock headings and "i" icons + text before Directives and More menu; it currently does not. 
> ![Image](https://github.com/user-attachments/assets/43ad79c6-d772-4abd-8b7b-90c539da1c0b)

- [ ] I will investigate this issue

> - All module stocks are not read by 508 screen reader; however, the title and the "i" description IS, which I think suffices.  Otherwise, it's basically stated twice. Screenshot below to clarify what I mean.
> <img width="570" alt="Image" src="https://github.com/user-attachments/assets/0d1dcf2f-6efc-4360-9207-bbd7551dc384" />

- I think I would agree that the behavior sounds sufficient

> - [ ] Medication Management radio buttons are not prefaced with the overall description like other modules with radio buttons. Screenshot below of the missing reader message:
> <img width="992" alt="Image" src="https://github.com/user-attachments/assets/e7ad77a3-6473-4fcc-b015-337597d8ec8a" />

- [ ] Good catch, this should be an easy fix

> - [ ] From the DDD>Team Data window>Upload button, removed text makes the yes/no buttons confusing; suggested improvement:
> ![Image](https://github.com/user-attachments/assets/298c99da-2579-45bd-88ee-691dc5eae40d)

- [ ] I might need to discuss this with you over a call later this week. I think I am a little confused on the issue.

> - [ ] There are no reader messages yet for feedback loops, decision meters, flow, episodes of care. These are cards #3302, #3303, #3304, #3305. How should this be prioritized moving forward? It would take a chunk of time (for me), then dev/you, etc.

- I think we might need to hold off on those for now. Right now I am prioritizing finishing the changes that we have started and I believe those cards were created in the backlog, but we did not start development on them.

> ℹ : I did not verify 508 compliance for Sim UI Play pages. I'm at my weekly limit of hours and can begin verifying those next week.

- @ljmoody I would hold off on testing those for now. They are outside of the scope of this card (which is only for the home page).

**Comment by lijenn** [comment]
#adjudicate_memo Dynamic Decision Dashboard was completed and in Prod for _MTL 3.7_.
